src/Storages/MergeTree/KeyCondition.h

```diff
@@ -278,6 +278,7 @@ class KeyCondition
 
     /// Checks that the index can not be used.
     bool alwaysUnknownOrTrue() const;
+    bool anyUnknownOrAlwaysTrue() const;
```

Comment:
Don't understand - missing comments.

Reply:
It means: check if the predicate is unknown anywhere or if it is always true?
Does it mean: check if the predicate is true or unknown but not false?
Motivation needs to be explained...